### PR TITLE
36 Unused state variables

### PR DESCRIPTION
**Pull request by decadanse**
closes <https://github.com/PrimeDAO/launch-contracts/issues/36>

here lint breaks
[conflicts resolved setup.seed.classes(0)](https://github.com/PrimeDAO/launch-contracts/pull/64/commits/19fd4cd22c77a6250f2f7d8494f9bb1df7ad5a4a) 

here and later package.json also edited and lint fixed
[fix?](https://github.com/PrimeDAO/launch-contracts/pull/64/commits/ea6392db590ddd252b18f638ff855857563530d8)

### CODE REVIEW

File: contracts/seed/Seed.sol [hunk 1]
```diff
@@ -32,7 +32,7 @@ contract Seed {
     uint256 public hardCap;
     uint256 public seedAmountRequired; // Amount of seed required for distribution
     uint256 public feeAmountRequired; // Amount of seed required for fee
-    uint256 public price; // price of a SeedToken, expressed in fundingTokens, with precision of 10**18
+    // uint256 public price; // price of a SeedToken, expressed in fundingTokens, with precision of 10**18
```
Comment (arseneeth) [inline contracts/seed/Seed.sol:35]: pls delete comments

File: contracts/seed/Seed.sol [hunk 2]
```diff
@@ -41,7 +41,7 @@ contract Seed {
     uint32 public vestingCliff;
     IERC20 public seedToken;
     IERC20 public fundingToken;
-    uint256 public fee; // Success fee expressed as a % (e.g. 10**18 = 100% fee, 10**16 = 1%)
+    // uint256 public fee; // Success fee expressed as a % (e.g. 10**18 = 100% fee, 10**16 = 1%)
```
Comment (arseneeth) [inline contracts/seed/Seed.sol:44]: same

File: contracts/seed/Seed.sol [hunk 3]
```diff
@@ -56,7 +56,7 @@ contract Seed {
     bool public minimumReached; // if the softCap[minimum limit of funding token] is reached
     bool public maximumReached; // if the hardCap[maximum limit of funding token] is reached
     bool public isWhitelistBatchInvoked; // if the whitelistBatch method have been invoked
-    uint256 public vestingStartTime; // timestamp for when vesting starts, by default == endTime,
+    // uint256 public vestingStartTime; // timestamp for when vesting starts, by default == endTime,
```
Comment (arseneeth) [inline contracts/seed/Seed.sol:59]: same

File: test/seed.js [hunk 4]
```diff
@@ -201,13 +201,13 @@ describe("Contract: Seed", async () => {
           expect((await setup.seed.softCap()).toString()).to.equal(
               softCap.toString()
           );
-          expect((await setup.seed.price()).toString()).to.equal(
-              price.toString()
-          );
+          // expect((await setup.seed.price()).toString()).to.equal(
```
Comment (arseneeth) [inline test/seed.js:204]: same

**Review by arseneeth — CHANGES_REQUESTED** [review]

File: test/seed.js (hunk 4 above)
Comment (decadanse) [reply test/seed.js]: Deleted all mentioned comments (all of the comments with deleted now parts)

**Review by arseneeth — APPROVED** [review]

**Pull request merged**